chainio/clients/avsregistry/reader.go

```diff
@@ -220,3 +369,75 @@ func (r *AvsRegistryChainReader) IsOperatorRegistered(
 	registeredWithAvs := operatorId != [32]byte{}
 	return registeredWithAvs, nil
 }
+
+func (r *AvsRegistryChainReader) QueryExistingRegisteredOperatorPubKeys(
+	ctx context.Context,
+	startBlock *big.Int,
+	stopBlock *big.Int,
+) ([]types.OperatorAddr, []types.OperatorPubkeys, error) {
+
+	blsApkRegistryAbi, err := abi.JSON(bytes.NewReader(eigenabi.BLSApkRegistryAbi))
+	if err != nil {
+		r.logger.Error("Error getting Abi", "err", err)
+		return nil, nil, err
+	}
+
+	query := ethereum.FilterQuery{
+		FromBlock: startBlock,
+		ToBlock:   stopBlock,
+		Addresses: []gethcommon.Address{
+			r.blsApkRegistryAddr,
+		},
+		Topics: [][]gethcommon.Hash{{blsApkRegistryAbi.Events["NewPubkeyRegistration"].ID}},
+	}
+
+	logs, err := r.ethClient.FilterLogs(ctx, query)
+	if err != nil {
+		r.logger.Error("Error filtering logs", "err", err)
+		return nil, nil, err
+	}
+	r.logger.Info("logs:", "logs", logs)
```

Comment:
instead of `logs:` - can we say specifically transactionLogs or something?

Reply:
[2c05b2c](https://github.com/Layr-Labs/eigensdk-go/pull/91/commits/2c05b2c4d9bc80813c80b94d900b161d2a11b539)